### DISCUSSION TITLE
Deployed link login not working

### DISCUSSION THREAD

**Discussion by elliewalkersm** [nss-evening-cohort-14/evening-client-side#263] [Help Me!] [answered]
Ellie Walker

Describe your issue
When on my deployed site, I tried to navigate to my login route, but I get the page not found error.

Have you searched the other help tickets? Yes.

I have tried:
re-deploying
adding my deployed link to firebase auth

what i've searched:
https://answers.netlify.com/t/support-guide-i-ve-deployed-my-site-but-i-still-see-page-not-found/125
https://stackoverflow.com/questions/50752350/page-not-found-when-trying-to-access-a-site-deployed-on-netlify

**Comment by TrinityChristiana** [comment]
Can you please add what you've researched?

**Comment by drteresavasquez** [comment]
Also, see this ticket: https://github.com/nss-evening-cohort-14/evening-client-side/discussions/248

**Reply by elliewalkersm** [reply]
I added that file and redeployed, but i'm still getting the page not found

**Comment by elliewalkersm** [comment, marked as answer]
I got it working! I added a file calle "_redirects" in the public folder and put this inside. 

/*    /index.html    200

I also changed my login.js to Login.js because Netlify needs capital files.
